tests/conftest.py

```diff
@@ -39,6 +63,25 @@ def docker(event_loop):
 
 
 @pytest.fixture
+def swarm(event_loop, docker):
```

Comment:
I'd like to keep tests as easy as possible, at least until we have a clear and shared opinion about how we should handle them.

Specifically why considering swarm like a fixture?